.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.wasm32-wasi]
+runner = 'enarx run'
```

Comment:
Although it's nice for local development, I don't think we should include it in this repo